### PR TITLE
Add exports fields to support dual export

### PR DESCRIPTION
**Pull request by tetsuharuohzeki**
https://github.com/karen-irc/option-t/issues/421

### CODE REVIEW

**Comment by saneyuki** [comment]
[webpack and rollup have not supported `exports` yet](https://github.com/karen-irc/option-t/issues/421#issuecomment-585760577).

We don't have a strong reason to hurry up to implement this.

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "cpx": "^1.5.0",
     "del-cli": "^3.0.1",
     "eslint": "^6.8.0",
+    "option-t": "/Volumes/Source/js/option-t/__dist",
```
Comment (tetsuharuohzeki) [inline package.json]: Remove this line.

File: yarn.lock [hunk 2]
```diff
@@ -3573,6 +3573,9 @@ onetime@^5.1.0:
   dependencies:
     mimic-fn "^2.1.0"
 
+option-t@/Volumes/Source/js/option-t/__dist:
+  version "22.3.2"
```
Comment (tetsuharuohzeki) [inline yarn.lock]: Remove this line.

File: tools/package_json_rewriter/transformer/add_exports_field/public_api.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        return {
+            'import': esm,
+            'require': cjs,
+            // 'default'
+        };
+    }
+}
+
+async function addPublicAPIToExportsFields(o) {
+    // TODO:
```
Comment (tetsuharuohzeki) [inline tools/package_json_rewriter/transformer/add_exports_field/public_api.js:39]: Remove this line.

**Pull request merged**